packages/providers/src/index.ts

```diff
@@ -8,4 +8,4 @@ export * from './resource';
 export { default as Provider } from './provider';
 export * from './transaction-request';
 export * from './transaction-response';
-export * from './util';
+export { calculatePriceWithFactor } from './util';
```

Comment:
now we have the arrayify inside the utils file. Exporting all would conflict with arrayify exported on the fuels package.

Reply:
Please, consider renaming the method to `arraifyFromUint8Array` or similar, to make the customization more evident. This will avoid conflicts while providing clarity to the intent of the customized `arraify`.